instrumentation/opentelemetry-instrumentation-django/README.rst

```diff
@@ -30,6 +30,19 @@ For example,
 
 will exclude requests such as ``https://site/client/123/info`` and ``https://site/xyz/healthcheck``.
 
+Request attributes
+********************
+To extract certain attributes from Django's request object and use them as span attributes, set the environment variable ``OTEL_PYTHON_DJANGO_TRACED_REQUEST_ATTRS`` to a comma
```

Comment:
Would be good to link to https://docs.djangoproject.com/en/3.1/ref/request-response/#attributes